### DISCUSSION TITLE
zoom tool

### DISCUSSION THREAD

**Discussion by theviper22** [obsproject/obs-studio#7515] [Development Questions] [answered]
It would be very helpful to be able to use zoom in the internal browser of obs, such as chrome, firefox etc.

**Comment by Fenrirthviti** [comment, marked as answer]
Please use the [Ideas page](https://ideas.obsproject.com/) for suggestions.

You can show your supposed for this specific request here: https://ideas.obsproject.com/posts/147/add-a-zoom-feature
